css/main.css

```diff
@@ -0,0 +1,59 @@
+body{
+    margin : 0;
+}
+.navigation ul li{
+    float: left;
+    list-style: none;
+    color: white;
+    padding:10px;
+    padding-top: 3px;
+   
+    
+    
+}
+.base{
+    text-decoration: underline;
+}
+.navigation{
+    background-color: #5e5e5e;
+    overflow:auto;
+}
+.navigation a {
+    text-decoration:none;
+    color: white;
+}
+
+
+
+.Section{
```

Comment:
it's better to name your css classes starting from lower case letter.

Reply:
Sure,i will do that